.golangci.yml

```diff
@@ -10,3 +10,6 @@ linters-settings:
     exclude: .errcheck_excludes.txt
   lll:
     line-length: 200
+  funlen:
```

Comment:
This linter [recently added](https://github.com/golangci/golangci-lint#september-2019) to `golangci-lint`, I've changed its parameters temporarily, otherwise, I need to introduce changes which are not in the scope of this PR.

Reply:
sgtm, feel free to follow up